### PR TITLE
Clean up and added presets file path to list parsers output #1662

### PR DESCRIPTION
**Pull request by joachimmetz**
Clean up and added presets file path to list parsers output #1662

### CODE REVIEW

File: plaso/cli/extraction_tool.py [hunk 1]
```diff
@@ -123,6 +127,24 @@ def _CreateProcessingConfiguration(self, knowledge_base):
 
     return configuration
 
+  def _GetParserPresetsInformation(self):
```
Comment (Onager) [inline plaso/cli/extraction_tool.py]: Please move this functionality to the ParserManager. There's already similar methods (GetParsersInformation) there for producing lists of tuples like this for parsers and plugins.

File: plaso/cli/extraction_tool.py [hunk 2]
```diff
@@ -186,6 +208,19 @@ def _PreprocessSources(self, extraction_engine):
 
     logger.debug('Preprocessing done.')
 
+  def _ReadParserPresetsFromFile(self):
```
Comment (Onager) [inline plaso/cli/extraction_tool.py]: ReadPresetsFromFile() can also result in a uncaught MalformedPresetError from _ReadPresetsFromFileObject() that would be nice to catch here and raise as BadConfigOption as well - please add this functionality.

File: plaso/cli/extraction_tool.py [hunk 3]
```diff
@@ -186,6 +208,19 @@ def _PreprocessSources(self, extraction_engine):
...
+    """Reads the parser presets from the presets.yaml file.
+
+    Raises:
+      BadConfigOption: if the parser presets file cannot be read.
+    """
+    self._presets_file = os.path.join(self._data_location, 'presets.yaml')
```
Comment (Onager) [inline plaso/cli/extraction_tool.py]: Please move 'presets.yaml' into a constant - _PRESET_FILE_NAME or similar.

File: plaso/cli/extraction_tool.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/cli/extraction_tool.py]: ack

File: plaso/cli/extraction_tool.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/cli/extraction_tool.py]: ack

File: plaso/cli/extraction_tool.py (hunk 3 above)
Comment (joachimmetz) [reply plaso/cli/extraction_tool.py]: ack

**Pull request merged**